docs/current_docs/api/snippets/functions/functions-complex/php/dagger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "my-module",
+  "engineVersion": "v0.15.3",
+  "sdk": "php",
+  "source": "."
+}
```

Comment:
@jedevc @kpenfound @jasonmccallister if we want to (eventually) make this a testable module, do we need anything other than these two files?

Reply:
@vikram-dagger In my local testing, I needed these files at a minimum to run the functions.

```
php-sample ➤ ls                                                                           git:main*
dagger.json    entrypoint.php src
php-sample ➤ cat dagger.json                                                              git:main*
{
  "name": "php-sample",
  "engineVersion": "v0.15.3",
  "sdk": "php",
  "source": "."
}
php-sample ➤ tree                                                                         git:main*
.
├── dagger.json
├── entrypoint.php
└── src
    └── PhpSample.php

2 directories, 3 files
```

Reply:
Updated with the minimum set of 3 files now

Reply:
> Updated with the minimum set of 3 files now

fixed my list above `entrypoint.php` now (not `.sh`)

Reply:
As per https://github.com/dagger/dagger/pull/9509#discussion_r1946405358, we should also include composer.json. I've added it for the examples in this PR and in https://github.com/dagger/dagger/pull/9509